analysed-packages/tcpdump/version-4.99.1/tcpdump-4.99.1-OSS-disclosure.txt

```diff
@@ -2617,16 +2614,75 @@ Yoyodyne, Inc., hereby disclaims all copyright interest in the program
 
 --------------------------------------------------------------------------------
 
-Dual-license
+BSD-4-Clause
+
+This product includes software developed by the Kungliga Tekniska Högskolan and
+its contributors.
+
+--------------------------------------------------------------------------------
+
+BSD-4-Clause
+
+This product includes software developed by Yen Yen Lim and
+North Dakota State University
+
+--------------------------------------------------------------------------------
+
+BSD-4-Clause
+
+This product includes software developed by the NetBSD
+Foundation, Inc. and its contributors.
+
+--------------------------------------------------------------------------------
+
+BSD-4-Clause-Shortened
+
+This product includes software developed by the University of California,
+Lawrence Berkeley Laboratory and its contributors.
+
+--------------------------------------------------------------------------------
+
+BSD-4-Clause-UC
+
+This product includes software developed by the University of
+California, Berkeley and its contributors.
+
+--------------------------------------------------------------------------------
+
+BSD-4-Clause-UC
+
+This product includes software developed by Bruce M. Simpson.
+
+--------------------------------------------------------------------------------
+
+BSD-4-Clause-UC
+
+This product includes software developed by the Computer Systems Engineering
+Group at Lawrence Berkeley Laboratory.
+
+--------------------------------------------------------------------------------
+
+LicenseRef-Dual-license
+
+To the extent that files may be licensed under either LicenseRef-BSD-4-Clause-
```

Comment:
Please change this to:
To the extent that files may be licensed under either GPL-1.0-or-later OR Permission-Notice, in this context,
Permission-Notice has been chosen. This choice shall not restrict the freedom of other users to select either
GPL-1.0-or-later OR Permission-Notice licenses. For convenience, all license texts are provided.